src/include/storage/storage_defs.h

```diff
@@ -45,6 +47,16 @@ STRONG_TYPEDEF(layout_version_t, uint32_t);
  * @warning If you change the layout please also change the way header sizes are computed in block layout!
  */
 struct alignas(common::Constants::BLOCK_SIZE) RawBlock {
+  /**
+   * Data Table for this RawBlock. This is used by indexes and GC to get back to the DataTable given only a TupleSlot
+   */
+  DataTable *data_table_;
+
+  /**
+   * Padding for flags or whatever we may want in the future.
```

Comment:
Probably should explicitly state that padding size is determined by `sizeof(layout_version_t)` below.  The typedef makes this logic not immediately obvious.